vcpkg.json

```diff
@@ -22,6 +42,45 @@
                     "default-features": false
                 }
             ]
+        },
+        "serialization": {
```

Comment:
To be honest I don't like the `serialization` name because it does not clearly communicate what it means (the description does not contain this word for example). Might be too disruptive to change though.

Reply:
I agree, but for now I'd prefer to match our existing feature names for clarity.